### DISCUSSION TITLE
Подписка...

### DISCUSSION THREAD

**Discussion by vanja-san** [cawa-93/anime-library#156] [Вопросы и ответы] [answered]
Я так понимаю, что бы смотреть аниме, нужна подписка на Шики?

**Comment by cawa-93** [comment, marked as answer]
🤔 "подписка на Шики"? Нет. Но нужна премиум подписка на сайте smotret-anime.online

**Reply by vanja-san** [reply]
Ну то есть нужна, понятно. Спасибо.
